kroxylicious-krpc-plugin/src/test/resources/Data/example-expected-FetchRequest.java.txt

```diff
@@ -107,7 +108,7 @@ public class FetchRequestData implements ApiMessage {
     /** @return the highest valid version for this message. */
     @Override
     public short highestSupportedVersion() {
-        return 15;
+        return 16;
```

Comment:
surely this should return the `HIGHEST_SUPPORTED_VERSION` constant and not hard code its own value